build

```diff
@@ -32,19 +32,25 @@ sleep 1.5
 # Recreate output folder
 rm -Rf docs
 mkdir -p docs/posts
+mkdir docs/posts/feed
 
 cp -r static docs/static
 cp CNAME docs/CNAME
 cp static/favicon.ico docs/favicon.ico
 cp static/robots.txt docs/robots.txt
 ./server --pages > docs/pages.txt
 ./server --posts > docs/posts.txt
+./server --feed > docs/rssfeed.txt
```

Comment:
Why is this needed? Cant you save the output of the feed endpoint to a file directly?